README.md

```diff
@@ -141,17 +141,7 @@ In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning
 
 ### Referencing
 
-To make it easier to reference these feature flags in code, we recommend to define feature flag variables like below.
-
-``` C#
-// Define feature flags using const strings
-public static class MyFeatureFlags
-{
-    public const string FeatureT = "FeatureT";
-    public const string FeatureU = "FeatureU";
-    public const string FeatureV = "FeatureV";
-}
-```
+In the following code examples, MyFeatureFlags.FeatureX which is a const string property of the MyFeatureFlags class, is used for referencing the name of feature flag "FeatureX".

```

Comment:
What do you think about just removing MyFeatureFlags altogether and using string literals?

Reply:
Sure. That's my original thought https://github.com/microsoft/FeatureManagement-Dotnet/pull/275#discussion_r1351194449

Reply:
Oh, sorry. Missed it.